docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
+
+| DIMM vendor | Part Number | Size | SPD profile | Configuration 1/2/4 |
+|:-----------:|:-----------:|:----:|:-----------:|:-------------------:|
+| Kingston    | [KF556C40BB/8][6] |8GB | 5600MT/s 40-40-40 1.25V | &#10004;/&#10004;/&#10004; |
+| Crucial     | [CT8G48C40U5.M4A1/8][7] |8GB | 4800MT/s 40-39-39 1.1V | &#10004;/&#10004;/&#10004; |
+| ADATA       | [AD5U48008G-S][9] |8GB | 4800MT/s 40-40-40 1.1V | &#10004;/&#10004;/&#10004; |
+
+[1]: https://www.kingston.com/dataSheets/KF436C17BBK4_32.pdf
+[2]: https://www.corsair.com/eu/en/Categories/Products/Memory/VENGEANCE-LPX/p/CMK16GX4M2B3200C16
+[3]: https://www.kingston.com/dataSheets/KF432C16BB1_16.pdf
+[4]: https://www.pny.com/anarchy-x-ddr4-red?sku=MD16GK2D4320016AXR
+[5]: https://www.kingston.com/datasheets/KF436C18BBK2_64.pdf
+[6]: https://www.kingston.com/memory/gaming/kingston-fury-beast-ddr5-memory
+[7]: https://www.crucial.com/memory/ddr5/ct2k16g48c40u5
+[8]: https://www.kingston.com/datasheets/HX426C16FRK4_64.pdf
+[9]: https://www.adata.com/us/consumer/dram-module-ddr5-4800-u-dimm
+[10]: https://www.amazon.com/OLOy-Memory-288-Pin-Desktop-MD4U323216DJDA/dp/B0836NZHWR
+
+=== "GPU HCL"
+
+    GPU Hardware Compatibility List presents the GPUs tested and verified
+    to work with Dasharo by community. The following list does not include GPU
+    which is tested and verfied in 3mdeb laboratory - this information might be
+    found in [Hardware Matrix](hardware-matrix.md) documentation.
+
+    Legend:
+
+    * GPU name: the full name of GPU including vendor and model name.
+    * Memory size: total amount of GPU memory declared by vendor.
+    * Memory type: GPU's type of memory.
+    * Bandwidth: GPU's bandwidth.
+    * PCI-E Architecture: declared by producer generation of PCI-E architecture.
+    * Multi-Graphics Technology: information about support for Multi-Graphics
+        Technology.
+
+    Information about GPU might be read from GPU package or documentation.
+
+    | GPU name         | Memory size | Memory type  | Bandwidth | PCI-E Gen | Multi-Graphics Technology | Results                |
+    |:----------------:|:-----------:|:------------:|:---------:|:---------:|:-------------------------:|:----------------------:|
+    | Nvidia GeForce GTX 1060   | 3072 MB  | GDDR5  | 192GB/s   | Gen3      | 1                         | [Qubes HCL reports][1] |
+    | MSI Radeon RX 6950 XT     | 16 GB    | GDDR6  | 576GB/s   | Gen4      | 1                         | |
+    | EVGA NVidia RTX 2080      | 8 GB     | GDDR6  | 448GB/s   | Gen3      | 1                         | |
+    | PNY NVidia RTX A5000      | 24 GB    | GDDR6  | 768GB/s   | Gen4      | 1                         | |
+    | Nvidia GeForce GTX 1080TI | 11264 MB | GDDR5X | x16       | Gen3      | 1                         | [Qubes HCL reports][2] |
+    | MSI Radeon RX 6500 XT MECH 2X 4G OC | 4 GB     | GDDR6  | x4 (x16 connector) | Gen4      | 1                         | Works only on Dasharo v1.1.0 or newer |
```

Comment:
Why `Works only on Dasharo v1.1.0 or newer` is in the column, where source of reports was supposed to be like? Version information is supposed to be in a dedidacted column, as with previous components. It is missing for GPU.